rest_framework/fields.py

```diff
@@ -178,7 +204,7 @@ def __init__(self, read_only=False, write_only=False,
         self.required = required
         self.default = default
         self.source = source
-        self.initial = self.initial if (initial is empty) else initial
+        self.initial = initial
```

Comment:
Before the change,  `initial = empty` from function signature, and `self.initial = None` (as defined).
After the change, if not passed in then `self.initial = None` and if passed in `self.initial = initial` - no change there.